src/main/java/eu/bbmri_eric/negotiator/database/model/Negotiation.java

```diff
@@ -168,6 +185,7 @@ public int hashCode() {
     return Objects.hash(getId());
   }
 
+  @Transactional
```

Comment:
This annotations should not be present in an entity

Reply:
Fixed in ffd04f6a2618411ceddbaaf038bd371a7633ad6f